sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/util/TimeSource.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.service.util;
+
+/** A simple interface capable of providing current time timestamps in millis and micros. */
+public interface TimeSource {
```

Comment:
What do we need this for? Did not found a usage in this PR? Would we need to depend on it, I hope not..

Reply:
Naah leave it, is fine, forgot about the writetime stuff. Is the query builder for grpc supporting this?